### ISSUE TITLE
Number field with "Limit numbers" disabled still has a min and max attribute on the front-end, breaking the field.

### ISSUE THREAD

**Issue by thijskaspers** [verbb/formie#635]
**Description**
A number field that is required, but with no further validation or "limit numbers" throws the validation error "please select a value that is no more than 0.". The HTML field is rendered with min="0" max="0" through the render form method.

<img width="361" alt="Schermafbeelding 2021-11-05 om 12 02 37" src="https://user-images.githubusercontent.com/1608184/140500928-878dd065-33d8-40fb-b1d1-8afd6229837a.png">

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.23
- Craft version: 3.7.18.2
- Multi-site: Yes

**Comment by engram-design** [comment]
This would probably be a case of values entered into the Min/Max values for the field (by enabling "Limit Numbers" entering values, and then disabling "Limit Numbers"). Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.23",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.24](https://github.com/verbb/formie/releases/tag/1.4.24)

**Comment by thijskaspers** [comment]
Thanks for the quick fix Josh! I think this bug occurred after the Freeform import we used, we didn't enable "limit numbers" manually, the fields were created by the Freeform tool in the Formie settings.
